lib/gilmour/backends/redis.rb

```diff
@@ -42,8 +45,8 @@ def setup_pubsub(opts)
       @subscriber = @publisher.pubsub_client
       register_handlers
     rescue Exception => e
-      GLogger.debug e.message
```

Comment:
The problem is when we enable the papertrail logging, it would be tough to restart gilmour based services. A genuine error should go out as an error.